packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -71,5 +71,8 @@
 
   "change_set_based_tools.enabled_description":
     "Specifies whether the commands for change-set-based development are available to the user.",
-  "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor"
+  "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor",
+  "force_source_retrieve_text": "SFDX: Retrieve Source from Org",
```

Comment:
This is the text for when a user right-clicks on a file path in the file explorer context. The file that the user clicks on could be a source file or a manifest file, so the message is more general.
![screen shot 2018-07-17 at 4 17 30 pm](https://user-images.githubusercontent.com/1639375/42848556-360cc5a6-89dd-11e8-9166-0d972ef90293.png)